webknossos-datastore/app/com/scalableminds/webknossos/datastore/services/DataServicesHolder.scala

```diff
@@ -8,14 +8,18 @@ import javax.inject.Inject
 /*
  * The BinaryDataService needs to be instantiated as singleton to provide a shared DataCubeCache.
  * The TracingStore one (for VolumeTracings) already is, since the surrounding VolumeTracingService is a singleton.
- * The DataStore one is singleton-ized via this holder.
+ * The DataStore is singleton-ized via this holder.
  */
 
-class BinaryDataServiceHolder @Inject()(config: DataStoreConfig) {
+class DataServicesHolder @Inject()(config: DataStoreConfig) {
 
   val binaryDataService = new BinaryDataService(
     Paths.get(config.Braingames.Binary.baseFolder),
     config.Braingames.Binary.loadTimeout,
     config.Braingames.Binary.cacheMaxSize)
 
+  val mappingService = new MappingService(
+    Paths.get(config.Braingames.Binary.baseFolder),
+    config.Braingames.Binary.mappingCacheMaxSize)
+
```

Comment:
I feel like the mapping service does not have to be in the holder, I’ll undo that and test if everything still works